morphsnakes/include/morphsnakes.h

```diff
@@ -20,15 +20,15 @@ struct Operator;
 template<>
 struct Operator<2>
 {
-    static constexpr OperatorDescriptor<4, 2> curvature  = {
-        {{0, 8},
-        {1, 7},
-        {2, 6},
-        {3, 5}}
+    static constexpr OperatorDescriptor<4, 2> curvature = {
+        { std::array<int, 2> {0, 8},
+          std::array<int, 2> {1, 7},
+          std::array<int, 2> {2, 6},
+          std::array<int, 2> {3, 5} }
```

Comment:
Is there any reason for explicitly adding the constructor `std:array`? The type `OperatorDescriptor` is defined in such a way that this should be the default behavior. Did you find any problem with it?